### PR TITLE
Improved comments in HexDumpBytes #390

### PR DESCRIPTION
**Pull request by peter-lawrey**
Also I updated the README examples

### CODE REVIEW

File: src/test/java/net/openhft/chronicle/wire/issue/JSON222Test.java [hunk 1]
```diff
@@ -119,6 +119,8 @@ public void testJSON() throws IOException {
                 if (expected.contains("\r\n"))
                     expected = expected.replaceAll("\r\n", "\n");
                 String actual = bytes2.toString();
+                // TODO FIX
```
Comment (JerryShea) [inline src/test/java/net/openhft/chronicle/wire/issue/JSON222Test.java]: should we have an issue for this?

Comment (peter-lawrey) [reply src/test/java/net/openhft/chronicle/wire/issue/JSON222Test.java]: #392

File: src/main/java/net/openhft/chronicle/wire/ValueOut.java [hunk 2]
```diff
@@ -967,6 +967,12 @@ default WireOut writeString(CharSequence x) {
         return text(x);
     }
 
+    default WireOut writeInt(IntConverter intConverter, int i) {
+        StringBuilder sb = Wires.acquireStringBuilder();
```
Comment (RobAustin) [inline src/main/java/net/openhft/chronicle/wire/ValueOut.java:971]: maybe use a different -> Wires.acquireStringBuilder() instance

**Review by RobAustin — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/wire/ValueOut.java (hunk 2 above)
Comment (peter-lawrey) [reply src/main/java/net/openhft/chronicle/wire/ValueOut.java:971]: Wires.acquireStringBuilder() is no longer public however Wires.acquireBytes() is #393

**Pull request merged**